### ISSUE TITLE
Some questions about the code

### ISSUE THREAD

**Issue by Zhangruiqi111** [HantaoShu/DeepSEM#14]
In Model.py，line 131 ：logits, prob, y = self.qyx(x.squeeze(2), temperature) ，I can't understand the specific meaning of logits, prob,y, I hope you can explain it，Thank you.
